spec.html

```diff
@@ -9,15 +10,16 @@
 contributors: Guy Bedford, Bradley Farias, Michael Ficarra
 </pre>
 
-<emu-clause id="sec-demo-clause">
-  <h1>This is an emu-clause</h1>
-  <p>This is an algorithm:</p>
-  <emu-alg>
-    1. Let _proposal_ be *undefined*.
-    1. If _proposal_ is ~accepted~, then
-      1. Let _stage_ be 0.
-    1. Else,
-      1. Let _stage_ be -1.
-    1. Return ? ToString(_proposal_).
-  </emu-alg>
+<emu-clause number="22.1.3" id="sec-properties-of-the-string-prototype-object">
+  <h1>Properties of the String Prototype Object</h1>
+
+  <emu-clause id="sec-string.prototype.iswellformed">
```

Comment:
and then
```suggestion
  <emu-clause id="sec-string.prototype.iswellformed" number="10">
```

Reply:
In that case, this should still have number 10, otherwise it'll be incorrectly marked as 22.1.3.1

Reply:
Sure, we can do that.